tests/rados/test_scrub_omap.py

```diff
@@ -0,0 +1,119 @@
+"""
+This file contains the  methods to verify the  OSD status during large omap entries.
+AS part of verification the script  perform the following tasks-
+   1. Creating omaps
+   2. Perform scrub and deepscrub
+   3. While creating objects check that osds status
+"""
+
+
+import datetime
+import time
+from threading import Thread
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.pool_workflows import PoolFunctions
+from utility.log import Log
+from utility.utils import method_should_succeed
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Test to create a large number of omap entries on the single PG pool and test osd resiliency
+    Returns:
+        1 -> Fail, 0 -> Pass
+    """
+    log.info(run.__doc__)
+    config = kw["config"]
+
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    rados_obj = RadosOrchestrator(node=cephadm)
+    pool_obj = PoolFunctions(node=cephadm)
+
+    pool_target_configs = config["verify_osd_omap_entries"]["configurations"]
+    omap_target_configs = config["verify_osd_omap_entries"]["omap_config"]
+
+    # Creating pools and starting the test
+    for entry in pool_target_configs.values():
+        log.debug(
+            f"Creating {entry['pool_type']} pool on the cluster with name {entry['pool_name']}"
+        )
+        if entry.get("pool_type", "replicated") == "erasure":
```

Comment:
Since we are creating just one replicated pool, Simple create_pool() call should be sufficient.

Reply:
Removed the code.